README.md

```diff
@@ -58,7 +58,36 @@ Because a fixed offset is supported, there is no need for a separate `OffsetDate
 ---------------------------------------------------------------------------------------------------
 
 # Scenario-Based Examples
-TBD
+
+### Convert a time in one time zone to a time at the same instant in another time zone.
```

Comment:
Yeah, this could probably be made clearer.

I was trying to differentiate between "convert 2:00 PST to 2:00 EST" and "convert 2:00 PST to 5:00 EST", and I'm not sure "Convert a time into a different time zone" communicates that distinction